pom.xml

```diff
@@ -101,17 +101,17 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
       </dependency>
       <dependency>
         <groupId>${project.groupId}</groupId>
-        <artifactId>score-client</artifactId>
+        <artifactId>score-server</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>${project.groupId}</groupId>
-        <artifactId>score-server</artifactId>
+        <artifactId>score-fs</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>${project.groupId}</groupId>
-        <artifactId>score-fs</artifactId>
+        <artifactId>score-client</artifactId>
         <version>${project.version}</version>
       </dependency>
```

Comment:
I don't know why this order was changed, but since we are updating it can we instead organize these alphabetically?

Reply:
Not sure why the order was changed, will organise it alphabetically and update the PR